rosetta/parallel/pandas_easy.py

```diff
@@ -142,6 +142,8 @@ def groupby_to_series_to_frame(
     # Set the index
     if hasattr(groupby_kwargs['by'], 'name'):
         indexname = groupby_kwargs['by'].name
+    elif isinstance(groupby_kwargs['by'], str):
```

Comment:
You should use `basestring` rather than `str` since `basesting` includes both standard and unicode strings...and some people (such as myself) do `from __future__ import unicode_literals`.